alembic/versions/b54de42ccd4c_create_issed_at_index_on_activity.py

```diff
@@ -0,0 +1,23 @@
+"""create_issed_at_index_on_activity
+
+Revision ID: b54de42ccd4c
+Revises: e52827be0601
+Create Date: 2020-07-27 13:01:25.768342
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'b54de42ccd4c'
+down_revision = 'e52827be0601'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("COMMIT;")
+    op.execute("CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_activity_issed_at ON activity(issued_at);")
```

Comment:
```suggestion
    op.create_index(op.f('idx_activity_issed_at'), 'activity', ['issued_at'], postgresql_concurrently=True)
```
Pour garder le même mécanisme que les autres migrations